src/lib.rs

```diff
@@ -62,8 +62,18 @@ pub use upper_camel::{
 
 use std::fmt;
 
+#[cfg(feature = "unicode")]
 use unicode_segmentation::UnicodeSegmentation;
```

Comment:
You can just move this import into `get_iterator`, right?

Reply:
Yes! great point.